Ginger/GingerCoreNET/ActionsLib/ActionsConversion/BusinessFlowConversionStatus.cs

```diff
@@ -0,0 +1,190 @@
+﻿using Amdocs.Ginger.Common.Enums;
+using GingerCore;
+using System.ComponentModel;
+using System.Drawing;
+
+namespace Amdocs.Ginger.CoreNET
+{
+    /// <summary>
+    /// This class is used to hold the wrapper for businessflow conversion status
+    /// </summary>
+    public class BusinessFlowConversionStatus : INotifyPropertyChanged
```

Comment:
change 'BusinessFlowConversionStatus' class name to 'BusinessFlowToConvert"

Reply:
changed